other/materials_designer/Introduction.ipynb

```diff
@@ -17,9 +17,9 @@
     "\n",
```

Comment:
<h4>3.1.3. Passing Data between notebooks</h4><p> </p><p>When running JupyterLab (Python) locally, the files only get stored in the "uploads" folder and are not sent to the outside runtime. In JupyterLite (Pyodide) environment, the files are sent to the outside runtime and stored in the "uploads" folder. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>